### PR TITLE
add support for binary heating cooling mode

### LINKED ISSUE #307: Gira climate control => single 1 bit for heating/cooling (no DPT 20.105)
**Description of problem:**
My climate room controllers (product details below) are not completely following the KNX standard. They correctly use DPT 20.102 for the operating modes which works fine in HA. 
However, the element does not support proper controller modes (no DPT 20.105) to e.g. switch between heating and cooling. Instead there is a single 1 bit to switch between heating and cooling mode (see function object 89 on manual page 56).
Reading the current value is even more tricky, since it is a single bit inside a 2 byte DPT 22.101 that needs to be read (see manual page 157).

=> Is there any way to get that to work currently? If not, what would be the best option to try to add support for that into the library? Sub parameters for `controller_status_address` and `controller_status_state_address` to control what DPT to use?

- [ ] using xknx standalone
- [X] using Home-Assistant knx plugin

**Version information:**
- xknx / Home-Assistant release with the issue: 0.11.3 / 0.112.0.dev0
- last working xknx / Home-Assistant release (if known):

**KNX installation:**
I have multiple installed Gira Push Button Sensor 3 Plus 2-gang (DE: KNX Tastsensor 3 Plus 2fach) = Order Number: 5142 00
They are a combined climate control and button/switch unit.

English manual: https://download.gira.de/data3/51423490.pdf

**Problem-relevant `xknx.yaml` or `configuration.yaml` entries:**
```YAML
climate:
  - platform: knx
    name: KNX Office
    temperature_address: '3/1/4'
    target_temperature_address: '3/2/4'
    target_temperature_state_address: '3/6/4'
    operation_mode_address: '3/4/2'
    operation_mode_state_address: '3/4/3'
    # controller_status_address: '3/4/0'
    # controller_status_state_address: '3/4/1'
    min_temp: 7.0
    max_temp: 32.0
```

3/4/0 and 3/4/1 are the GAs for the 1 bit mode switch and the 2 byte status reading.

### PR DESCRIPTION
**Pull request by farmio**
This adds support for binary heating cooling mode

Heating mode defaults to DPT1 value 1
Cooling mode defaults to DPT1 value 0
as used by Gira, Jung and MDT actuators.

closes #307

### CODE REVIEW

File: xknx/remote_value/remote_value_climate_mode.py [hunk 1]
```diff
@@ -121,3 +130,24 @@ def from_knx(self, payload):
             return None
         raise CouldNotParseTelegram("payload invalid",
                                     payload=payload, device_name=self.device_name, feature_name=self.feature_name)
+
+
+class RemoteValueBinaryHeatCool(_RemoteValueBinaryClimateMode):
```
Comment (marvin-w) [inline xknx/remote_value/remote_value_climate_mode.py:135]: Do we need a test for this new class?

Comment (farmio) [reply xknx/remote_value/remote_value_climate_mode.py:135]: Right! 
I miss the coveralls bot nagging at me for not writing tests 🤖

Comment (marvin-w) [reply xknx/remote_value/remote_value_climate_mode.py:135]: Why does it not work anymore?

Comment (farmio) [reply xknx/remote_value/remote_value_climate_mode.py:135]: I think netlify just checks PRs to the main branch. 
Coveralls is gone for some time. I don't know what happened there.

**Pull request merged**